how-to/software/package-management.md

```diff
@@ -245,88 +245,6 @@ deb-src http://security.ubuntu.com/ubuntu DISTRO-SHORT-CODENAME-security univers
 deb http://security.ubuntu.com/ubuntu DISTRO-SHORT-CODENAME-security multiverse
 deb-src http://security.ubuntu.com/ubuntu DISTRO-SHORT-CODENAME-security multiverse
 ```
-<h2 id='heading--automatic-updates'>Automatic updates</h2>
```

Comment:
Thanks for splitting the commits - that made it really easy to read.
I like all the changes, but feel we should leave more traces to the moved content.
Adding it to the index is fine, but could we leave a link to the automatic upgrades section - either directly here where this has been removed or the bit below which already has "Further reading" ?

Reply:
I'd put it here, if this is where it was before, this is the logical place people will expect to find it. You could even keep the heading if you wanted, with the original line 250 to give context (with a link to the new page).

Reply:
How about my latest update?

Reply:
lgtm :) thanks